src/middleware/devtools.ts

```diff
@@ -146,6 +149,22 @@ const extractConnectionInformation = (
   return { type: 'tracked' as const, store, ...newConnection }
 }
 
+const removeStoreFromTrackedConnections = (
+  name: string | undefined,
+  store: string | undefined,
+) => {
+  if (store === undefined) return
```

Comment:
If `store` is undefined, we don't do anything.
So, is the goal of this PR is to avoid memory leaks?